app/components/jobseekers/school_group_overview_component.html.haml

```diff
@@ -36,7 +36,7 @@
   -# TODO: Unable to show the map as we cannot extract geolocation data from GIAS
```

Comment:
Can we turn this TODO into done now?

Reply:
School groups still don't have geolocations yet, but I can uncomment the map for if/when they do.

Reply:
We should make a ticket to give them geolocations then, because there isn't one at the minute

Reply:
I'll do it.

Reply:
https://dfedigital.atlassian.net/browse/TEVA-1297